### PR TITLE
Created baseFiles for README.md, main.nf, nextflow.config, gadi.config

### PR DESCRIPTION
**Pull request by nandan75**
Created baseFiles for README.md, main.nf, nextflow.config, gadi.config

ChingYu and Nandan will start with these files

### CODE REVIEW

**Comment by georgiesamaha** [comment]
Hi @nandan75, please unassign me and work with @chlu0571 to do reviews. I will review once workflow has been finalised and documentation has been written.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,244 @@
...
+        --NCPUS ${NCPUS} \
+       --refFasta /path/to/ref.fasta --refGtf /path/to/ref.gtf
+```
+
+Before running the pipeline you will need to load Nextflow and Singularity, both of which are globally installed modules on Gadi. You can do this by running the commands below:
+
+```
+module purge
+module load nextflow singularity
+```
```
Comment (chlu0571) [inline README.md:176]: do we need to specify the version of nextflow and singularity? Also, I load java as well.

**Review by chlu0571 — APPROVED** [review]
All good. I will base on these documents and work on the fastqc /  multiqc and trim

File: README.md (hunk 1 above)
Comment (nandan75) [reply README.md:176]: Check with loading the above as is 
`module load nextflow singularity` 
If it does not run the pipeline, you can load nextflow version. 

By doing the above you are loading the default (possibly latest) versions

**Pull request merged**